.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,42 @@
+name: Feature request
+description: Suggest an idea for this project.
+labels: [feature-request, triage-required]
+
+body:
+- type: checkboxes
+  attributes:
+    label:  Before logging a feature request please confirm the following
+    options:
+    - label: This is not a Security Disclosure, otherwise please follow the guidelines in [Security Policy](https://github.com/adobe/aepsdk-edge-ios/security/policy).
+      required: true
+    - label: I have searched in this repository's issues to see if it has already been reported.
+      required: true
+
+- type: textarea
+  id: description
+  attributes:
+    label: Feature request summary
+    description: Please provide a summary of the feature you request.

```

Comment:
wording: "Please provide a summary of the feature you are requesting." Or simply, "Please provide a summary of the feature."